lib/checks.js

```diff
@@ -246,4 +247,11 @@ function _hexFromByteArray(byteArray) {
     }
   }
   return out;
-};
\ No newline at end of file
+};
+
+function getCaseInsensitiveKey(obj, name) {
+  var realName = _.findKey(obj, function (value, key) {
```

Comment:
@AnthonyRonning We should probably do that in pure JS. Adding the `underscore` library to the bundle just for that seems like a bit much.

Reply:
https://gist.github.com/DaveAtDog/11207429

Reply:
Thanks for that link, altered it a bit so it could be used to find case insensitive keys as well. Removed underscore.

Reply:
@AnthonyRonning thanks for adding it!